File: java/hargabarang.java
import java.util.Scanner;

public class hargabarang {
    public static void main(String[] args) {
       try (Scanner input = new Scanner(System.in)) {;

        System.out.print("Masukkan nama barang: ");
        String nama = input.nextLine();

        System.out.print("Masukkan harga barang: ");
        double harga = input.nextDouble();

        System.out.print("Masukkan jumlah beli: ");
        int jumlah = input.nextInt();

        double total = harga * jumlah;
        double diskon = 0;

        // Logika diskon
        if (total > 17400) {
            diskon = total * 0.04; // Diskon 4%
        } else {
            diskon = 0; // Tidak ada diskon
        }

        double totalBayar = total - diskon;

        // Output struk
        System.out.println("\n==== STRUK PEMBELIAN ====");
        System.out.println("Nama Barang   : " + nama);
        System.out.println("Harga Satuan  : " + String.format("%.2f", harga));
        System.out.println("Jumlah Beli   : " + jumlah);
        System.out.println("Total Harga   : " + String.format("%.2f", total));
        System.out.println("Diskon        : " + String.format("%.2f", diskon));
        System.out.println("Total Bayar   : " + String.format("%.2f", totalBayar));
        System.out.println("==========================");
    }


    }
}

File: python/hargabarang.py
nama = input("masukan nama barang:")
harga = float(input("masukan harga barang:"))
jumlah = int(input("masukan jumlah beli:"))

total_harga = harga * jumlah 

if jumlah > 80000:
    diskon = total_harga * 0.10
else:
    diskon = 0
    
total_bayar = total_harga - diskon

print("\n ===== STRUK PEMBELIAN =====")
print("Nama Barang:    :    {nama}")
print(f"harga satuan   :     {harga}")
print(f/"Jumlah Beli   :     {jumlah}")
print(f"Total          :     {total_}")
print(f"Diskon         :     {diskon}")
print(f"Total Bayar    :     {total_bayar}")
print("===============================")
    
